functions/services/sendgrid/index.test.js

```diff
@@ -0,0 +1,57 @@
+const { addEmailToList } = require("./index");
+
+const faker = require("faker");
+
+jest.mock("@sendgrid/client");
+const client = require("@sendgrid/client");
+
+describe("SendgridService", () => {
+    beforeAll(() => {
+        process.env = {
+            SENDGRID_API_KEY: faker.random.uuid()
+        };
```

Comment:
Please Backup `process.env` and restore it after each test so it won't hurt other tests

like that:

```js

const orgEnv = {...(process.env)};
beforeEach(() => {
    process.env.SENDGRID_API_KEY = faker.random.uuid();
}
afterEach(() => {
    process.env = {...orgEnv};
}

```

Reply:
Cool, this makes sense. I originally felt that the `process.env` should not be a dependency on the tests and thus I just had things written over on each test suite would be fine. However, you do bring up a good point that it might not be the case for every test that will and currently exist.

Reply:
Updated